assert/assert.go

```diff
@@ -90,6 +90,14 @@ func (assert Assert) EqFloat(actual, expected float64) {
 	}
 }
 
+// EqFloat fails the test if two floats aren't equal. NaNs are considered equal.
+func (assert Assert) EqApproximate(actual, expected, epsilon float64) {
+	delta := actual - expected
```

Comment:
Can probably `math.Abs()` this.

Why don't we just add this to `EqFloat()` with epsilon 1e-10? Don't think we really lose anything, and this ends up being what it's used for anyways (comparing floats).